lib/bemxjst/index.js

```diff
@@ -519,7 +524,8 @@ BEMXJST.prototype.exportApply = function exportApply(exports) {
 
   // Add templates at run time
   exports.compile = function compile(templates) {
-    return self.compile(templates);
+    self.compile(templates);
+    return exports;
```

Comment:
Nah, just lack a way to the same as original compile — make an instance with templates in runtime or get templates from bundled one.